<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigiwagePlatform</class>
 <widget class="QWidget" name="DigiwagePlatform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="headLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>8</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Digiwage Platform</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>611</width>
     <height>88</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>721</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="useraddressLabel">
      <property name="text">
       <string>User public address</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="userLabel">
      <property name="text">
       <string>User identification name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="addressEdit">
      <property name="toolTip">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter User Address Registered in Platform</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="usernameEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="addressBookButton">
      <property name="maximumSize">
       <size>
        <width>23</width>
        <height>100000</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Choose an address from the address book</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
      </property>
      <property name="shortcut">
       <string>Alt+A</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="userButton">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="userstatusLabel">
      <property name="text">
       <string>User status</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="userstatusLabel2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>OFFLINE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
